src/chains/ethereum/ethereum/src/forking/trie.ts

```diff
@@ -42,44 +37,35 @@ export class ForkTrie extends GanacheTrie {
   private isPreForkBlock = false;
   private forkBlockNumber: bigint;
   public blockNumber: Quantity;
-  private metadata: CheckpointDB;
-  private db: LevelDB | UpgradedLevelDown;
+  private checkpointedMetadata: CheckpointDB;
+  /** The underlying database for `checkpointedMetadata */
```

Comment:
The `.db` property of a `CheckpointDB` strips the types so that we can't iterate over the whole database, so now we store the metadata's database separately from the checkpointed version. This is similar to the added `.db` property of the `GanacheTrie`